metadata-ingestion/examples/library/lineage_dataset_transform_with_query_text.py

```diff
@@ -0,0 +1,25 @@
+from datahub.metadata.urns import DatasetUrn
+from datahub.sdk.lineage_client import LineageClient
+from datahub.sdk.main_client import DataHubClient
+
+client = DataHubClient.from_env()
+lineage_client = LineageClient(client=client)
+
+sql_query = """
+SELECT 
+    customer_id,
```

Comment:
make this be dummy python code - because for sql, we want people to use the other method instead

Reply:
we should make people use `lineage_from_sql` when they have REAL SQL statement 
this should be an example of e.g. airflow DAG, etc